api-website/index.rst

```diff
@@ -10,7 +10,20 @@ Welcome to SalesGPT's documentation!
    :maxdepth: 2
    :caption: Contents:
 
+   salesgpt/salesgpt.agents
```

Comment:
@chemik-bit Can You please make the names more readable (eg `salesgpt.chains` -> Chains etc

Reply:
Np, done!